dvc/command/pkg.py

```diff
@@ -45,45 +114,105 @@ def add_parser(subparsers, parent_parser):
-        nargs="?",
-        default=".",
-        help="Target directory to deploy package outputs. "
-        "Default value is the current dir.",
+    pkg_install_parser.set_defaults(func=CmdPkgInstall)
+
+    PKG_UNINSTALL_HELP = "Uninstall package(s)."
+    pkg_uninstall_parser = pkg_subparsers.add_parser(
+        "uninstall",
+        parents=[parent_parser],
+        description=append_doc_link(PKG_UNINSTALL_HELP, "pkg-uninstall"),
+        help=PKG_UNINSTALL_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
     )
-    pkg_install_parser.add_argument(
-        "-s",
-        "--select",
-        metavar="OUT",
-        action="append",
-        default=[],
-        help="Select and persist only specified outputs from a package. "
-        "The parameter can be used multiple times. "
-        "All outputs will be selected by default.",
+    pkg_uninstall_parser.add_argument(
+        "targets", nargs="*", default=[None], help="Package name."
     )
-    pkg_install_parser.add_argument(
+    pkg_uninstall_parser.set_defaults(func=CmdPkgUninstall)
+
+    PKG_ADD_HELP = "Add package."
+    pkg_add_parser = pkg_subparsers.add_parser(
+        "add",
+        parents=[parent_config_parser, parent_parser],
+        description=append_doc_link(PKG_ADD_HELP, "pkg-add"),
+        help=PKG_ADD_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    pkg_add_parser.add_argument("name", nargs="?", help="Package name.")
+    pkg_add_parser.add_argument("url", help="Package URL.")
+    pkg_add_parser.add_argument(
         "-f",
-        "--file",
-        help="Specify name of the stage file. It should be "
-        "either 'Dvcfile' or have a '.dvc' suffix (e.g. "
-        "'prepare.dvc', 'clean.dvc', etc). "
-        "By default the file has 'mod_' prefix and imported package name "
-        "followed by .dvc",
+        "--force",
+        action="store_true",
+        default=False,
+        help="Force overwriting existing package config.",
     )
-    pkg_install_parser.set_defaults(func=CmdPkgInstall)
+    pkg_add_parser.set_defaults(func=CmdPkgAdd)
+
+    PKG_REMOVE_HELP = "Remove package."
+    pkg_remove_parser = pkg_subparsers.add_parser(
+        "remove",
+        parents=[parent_config_parser, parent_parser],
+        description=append_doc_link(PKG_REMOVE_HELP, "pkg-remove"),
+        help=PKG_REMOVE_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    pkg_remove_parser.add_argument("name", help="Package name.")
+    pkg_remove_parser.set_defaults(func=CmdPkgRemove)
+
+    PKG_MODIFY_HELP = "Modify package."
+    pkg_modify_parser = pkg_subparsers.add_parser(
+        "modify",
+        parents=[parent_config_parser, parent_parser],
+        description=append_doc_link(PKG_MODIFY_HELP, "pkg-modify"),
+        help=PKG_MODIFY_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    pkg_modify_parser.add_argument("name", help="Package name.")
```

Comment:
I noticed you can provide a name that doesn't yet exist in the config file and `modify` goes on to create it. So you can in effect add a remote with `dvc pkg modify {new pkg name} url {pkg url}`. Should this condition be validated?

Reply:
(Same for the `option` argument.)